vault/auth.go

```diff
@@ -159,13 +160,25 @@ func (v *Vault) UserPassLogin() string {
 func (v *Vault) EC2Login() string {
 	role := env.Getenv("VAULT_AUTH_AWS_ROLE")
 	mount := env.Getenv("VAULT_AUTH_AWS_MOUNT", "aws")
+	nonce := env.Getenv("VAULT_AUTH_AWS_NONCE")
+	output := env.Getenv("VAULT_AUTH_AWS_NONCE_OUTPUT")
+	perms := env.Getenv("VAULT_AUTH_AWS_NONCE_PERMISSION", "0666")
```

Comment:
I'm not sure I'm a fan of `0666` as a default permission since it'll be group/other read/writable. IMO `0600` is a safer default.

Also is there even a need for this to be configurable? I'm not sure I really see a use-case where someone would need to save a nonce that's world-readable, and can't simply `chmod` it after `gomplate` runs...

Reply:
I can change the default or make it non-configurable.

Reply:
ok, I'd rather make it explicitly `0600` lower down, and also remove the option to configure

Reply:
Done